server/models/Order.js

```diff
@@ -288,11 +289,11 @@ function defineModel() {
   Order.prototype.getOrCreateMembers = async function () {
     // Preload data
     this.collective = this.collective || (await this.getCollective());
-
-    // Register user as collective backer
+    const tier = await models.Tier.findOne({ where: { id: this.TierId } });
```

Comment:
:no_entry_sign: `this.TierId` can be null

Reply:
@Betree : I've added a null check for this case so that if the `TierID` is null we don't take that into account. Let me know if you see any issues with this approach. 🤔